Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,9 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+const isValidPass = require('./password-validator');
+
+test("Is prime number", function(){
```

Comment:
Are we testing prime number ?

Reply:
Not really, i failed to change the description on the test.